### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by eabase** [ageitgey/show-facebook-computer-vision-tags#64]
I tried this on Brave (Chrome) browser and (new) Fb layout, but it dosn't do anything. 

Does this still work?

**Comment by HiGein** [comment]
No. Facebook doesn't have these tags in the source code anymore.
